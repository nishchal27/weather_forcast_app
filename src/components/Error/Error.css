.Error {
  color: var(--error-color);
  font-weight: 500;
  text-transform: capitalize;
};
