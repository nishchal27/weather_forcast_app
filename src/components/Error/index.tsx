import * as React from 'react';

import './Error.css';

type Props = {
  error: string;
};

const Error: React.FC<Props> = ({ error }) =>
  <span className='Error'>{error}</span>;

export default Error;
