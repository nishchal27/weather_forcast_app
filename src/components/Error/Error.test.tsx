import React from 'react';
import { shallow } from 'enzyme';

import Error from './';

describe('<Error />', () => {
  it('renders without crashing', () => {
    const error = 'Something went wrong';
    const wrapper = shallow(<Error error={error}/>);

    expect(wrapper).toMatchSnapshot();
  });
});
