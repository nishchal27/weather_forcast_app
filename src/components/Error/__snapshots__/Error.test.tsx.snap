// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> renders without crashing 1`] = `
<span
  className="Error"
>
  Something went wrong
</span>
`;
